apps/api/v2/src/ee/bookings/2024-08-13/bookings.repository.ts

```diff
@@ -169,4 +169,33 @@ export class BookingsRepository_2024_08_13 {
       },
     });
   }
+
+  async getAcceptedUserBookingForTimeRange(userId: number, startTime: Date, endTime: Date) {
+    return this.dbRead.prisma.booking.findFirst({
+      where: {
+        userId,
+        status: "ACCEPTED",
+        startTime: {
+          gte: startTime,
+          lt: endTime,
+        },
+      },
+    });
+  }
+
+  async getBookingsAsAttendeeWithinTimeRange(userEmail: string, startTime: Date, endTime: Date) {
```

Comment:
Ah, it returns just 1 booking - so need to call it `getBookingAsAttendeeWithinTimeRange`